src/constructs/webcrawler/web-crawler-step-lambdas.ts

```diff
@@ -69,7 +72,10 @@ export default class WebCrawlerStepLambdas extends Construct {
     // Lambda for reading queued urls from the context table
     const readQueuedUrls = buildLambda('ReadQueuedUrlsLambda', 'readQueuedUrlsHandler');
     props.historyTable.grantReadWriteData(readQueuedUrls);
+    props.workingBucket.grantReadWrite(readQueuedUrls);
     readQueuedUrls.addToRolePolicy(props.createContextTablePolicy(['Query']));
+    readQueuedUrls.addEnvironment("STATE_MACHINE_URL_THRESHOLD", "100000");
+    readQueuedUrls.addEnvironment("PARALLEL_URLS_TO_SYNC", "1000");
```

Comment:
I like having the constants loaded by the lambdas via environment variables, but can we still have the definitions be in a constants file which explains what the constants to make it a bit easier for users to find how to adjust these?

I think it'd be best to:

* Introduce a new file `src/constants.ts` which has the comments about the constants and what they mean, as well as the values used in the samples.
* Update the lambdas to read directly from the env - fine if you want to abstract this away as in the `src/lambdas/constants.ts` file but let's call it `src/lambdas/env.ts` instead.
* Update the CDK code to use the constants defined in `src/constants.ts`

Reply:
Implemented the following:
- Deleted `src/lambda/config/constants.ts` as the constants will be loaded by the Lambdas via environment variables
- Added the constants to `src/constructs/webcrawler/constants.ts` with comments explaining these constants.
- Updated CDK code to use these constants as default values for the Lambdas' environment variables.
- Added `src/lambda/utils/env.ts` and `getEnvVariableAsInteger` util function.